tests/SprykerTest/Zed/EventBehavior/Business/EventBehaviorFacadeTest.php

```diff
@@ -116,6 +119,122 @@ function ($eventName, TransferInterface $eventTransfer) {
         $this->eventBehaviorFacade->triggerLostEvents();
     }
 
+    /**
+     * @return void
+     */
+    public function testExecuteResolvedPluginsBySources(): void
+    {
+        $behaviorStatus = Config::get(EventBehaviorConstants::EVENT_BEHAVIOR_TRIGGERING_ACTIVE, false);
+        if (!$behaviorStatus) {
```

Comment:
What are we checking with this case?

Reply:
Can\`t say for sure, this is common verification for event triggering methods in facade. I just copied one and change the part which is differs in my case (another trigger event method name, container dependencies )